lib/_stylesheets/app/_cookie-banner.scss

```diff
@@ -0,0 +1,12 @@
+/* ==========================================================================
+   #COOKIE BANNER
+   ========================================================================== */
+
+// Container width variable
+#nhsuk-cookie-banner .app-width-container {
```

Comment:
This could be added to _container scss file?

Reply:
I don't even know why we have it tbh.

this is _container.scss

```css
.app-width-container,
.app-breadcrumb .nhsuk-width-container {
  max-width: $service-manual-container-size;

  @media (min-width: 767px) and (max-width: 1164px) {
    margin: 0 nhsuk-spacing(5);
  }
}
```

Doesn't this mean its already covered by the `.app-width-container`?

Reply:
Think we should be, might need to remove it and check